### PR TITLE
API voters votes field value changed from count of voters to vote weight - Closes #2968

### LINKED ISSUE #2968: API voters votes field value changed from count of voters to vote weight
### Expected behavior
The `votes` field should have the count of voters
https://node01.lisk.io/api/voters?username=korben3&offset=0&limit=10&sort=publicKey%3Aasc
Even though the changes are valid, as the `votes` should reflect the vote weight, the breaking changes in a minor release is not allowed, so it needs to be the same as before, which is the count of voters.
### Actual behavior
`"votes"` field shows the vote weight of the delegate
https://testnet.lisk.io/api/voters?username=korben3&offset=0&limit=10&sort=publicKey%3Aasc
### Steps to reproduce
https://testnet.lisk.io/api/voters?username=korben3&offset=0&limit=10&sort=publicKey%3Aasc
### Which version(s) does this affect? (Environment, OS, etc...)
1..5.0

### PR DESCRIPTION
**Pull request by lsilvs**
### What was the problem?
Wrong value for votes in `/api/voters`

### How did I fix it?
I used `storage.entities.Account.count` to get the total accounts that voted for queried delegate

### How to test it?
GET `/api/voters?username=genesis_1&limit=1`

### Review checklist

* The PR resolves #2968
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: api/controllers/voters.js [hunk 1]
```diff
@@ -126,6 +125,8 @@ VotersController.getVoters = async function(context, next) {
 			_.pick(voter, ['address', 'publicKey', 'balance'])
 		);
 
+		data.votes = voters.length;
```
Comment (4miners) [inline api/controllers/voters.js]: It should be a total number of voters, not just length of the currently returned set. For example, if we specify `limit=1` but delegate have a total of `8000` voters it should still return `8000` here. Will it work like this?

Comment (lsilvs) [reply api/controllers/voters.js]: oh! Got it. It won't. Gonna fix it.

Comment (lsilvs) [reply api/controllers/voters.js]: @4miners  That's why the test was testing `to.be.least`, right?

Comment (4miners) [reply api/controllers/voters.js]: @4miners Looks like the test was not reliable. We should also update the description of this field in `swagger.yml` to match the current behavior.

Comment (lsilvs) [reply api/controllers/voters.js]: Ok. I will update swagger as well.

**Comment by lsilvs** [comment]
@4miners Not sure how to improve this test as the number of votes will change each time the test runs. Any idea on how to make it more reliable?

**Comment by 4miners** [comment]
@lsilvs Maybe you can keep calling API with increasing offset until it returns no results and then you will have the count?

File: test/functional/http/get/voters.js [hunk 2]
```diff
@@ -237,6 +237,32 @@ describe('GET /api/voters', () => {
...
+				let votesCount = 0;
+
+				// The following loop is a workaround to get votes count as the number of votes will change each time the test runs
+				// REF.: https://github.com/LiskHQ/lisk/pull/2969
+				do {
+					// eslint-disable-next-line no-await-in-loop
+					const result = await votersEndpoint.makeRequest(
+						{
+							username: validVotedDelegate.delegateName,
+							limit: 1,
```
Comment (4miners) [inline test/functional/http/get/voters.js]: We don't really need to pull only `1` vote per iteration, we can use max limit here.

**Pull request merged**